subprojects/wrapper/src/main/java/org/gradle/wrapper/Install.java

```diff
@@ -71,18 +89,28 @@ public File call() throws Exception {
 
                 verifyDownloadChecksum(configuration.getDistribution().toString(), localZipFile, distributionSha256Sum);
 
-                logger.log("Unzipping " + localZipFile.getAbsolutePath() + " to " + distDir.getAbsolutePath());
                 unzip(localZipFile, distDir);
 
                 File root = getAndVerifyDistributionRoot(distDir, safeDistributionUrl.toString());
                 setExecutablePermissions(root);
                 markerFile.createNewFile();
 
+                printReleaseSummary();
+
                 return root;
             }
         });
     }
 
+    private void printReleaseSummary() {
```

Comment:
:x: A null value for `gradleVersion` should be verified in `InstallTest.groovy`.